frontend/src/app/main/ui/shapes/frame.cljs

```diff
@@ -65,17 +65,17 @@
         render-id (mf/use-ctx muc/render-id)]
 
     [:*
-     [:g {:clip-path (when (not show-content) (frame-clip-url shape render-id))}
-      [:& frame-clip-def {:shape shape :render-id render-id}]
+     [:g {:key "clip-path" :clip-path (when (not show-content) (frame-clip-url shape render-id))}
```

Comment:
I don't think this are adding much value if they are not inside a loop

Reply:
true, I've removed this